cartridge/confapplier.lua

```diff
@@ -187,6 +187,18 @@ local function validate_config(clusterwide_config, _)
         conf_old = vars.clusterwide_config:get_readonly()
     end
     if conf_old == nil then
+        local instance_uuid = topology.find_server_by_uri(
+            conf_new.topology, vars.advertise_uri
+        )
+        if instance_uuid == nil then
```

Comment:
When instance_uuid is not found it's the first case. Is it possible  `instance_uuid  ~= box.info.uuid` (for some reason, e.g. when user makes some manipulations by hand)?

Reply:
No, it's not. `conf_old == nil` means the instance isn't configured yet. Even manual config editing wouldn't have such effect. Only chopping the code in tiny pieces would.